repl/src/main/scala/com/cloudera/livy/repl/SparkInterpreter.scala

```diff
@@ -48,6 +50,7 @@ class SparkInterpreter(conf: SparkConf) extends Interpreter {
   private val outputStream = new ByteArrayOutputStream()
   private var sparkIMain: SparkIMain = _
   private var sparkContext: SparkContext = _
+  private var sqlContext: SQLContext = _
```

Comment:
It's probably a good idea to set this variable to `null` in the `close` method (since `SQLContext` doesn't have a stop or close or anything).